bot/cogs/snakes.py

```diff
@@ -0,0 +1,1231 @@
+import asyncio
+import colorsys
+import logging
+import os
+import random
+import re
+import string
+import textwrap
+import urllib
+from functools import partial
+from io import BytesIO
+from typing import Any, Dict
+
+import aiohttp
+import async_timeout
+from discord import Colour, Embed, File, Member, Message, Reaction
+from discord.ext.commands import AutoShardedBot, BadArgument, Context, bot_has_permissions, command
+from PIL import Image, ImageDraw, ImageFont
+
+from bot.constants import ERROR_REPLIES, OMDB_API_KEY, SITE_API_KEY, SITE_API_URL, YOUTUBE_API_KEY
+from bot.converters import Snake
+from bot.decorators import locked
+from bot.utils.snakes import hatching, perlin, perlinsneks, sal
+
+log = logging.getLogger(__name__)
+
+# region: Constants
+# Color
+SNAKE_COLOR = 0x399600
+
+# Antidote constants
+SYRINGE_EMOJI = "\U0001F489"  # :syringe:
+PILL_EMOJI = "\U0001F48A"     # :pill:
+HOURGLASS_EMOJI = "\u231B"    # :hourglass:
+CROSSBONES_EMOJI = "\u2620"   # :skull_crossbones:
+ALEMBIC_EMOJI = "\u2697"      # :alembic:
+TICK_EMOJI = "\u2705"         # :white_check_mark: - Correct peg, correct hole
+CROSS_EMOJI = "\u274C"        # :x: - Wrong peg, wrong hole
+BLANK_EMOJI = "\u26AA"        # :white_circle: - Correct peg, wrong hole
+HOLE_EMOJI = "\u2B1C"         # :white_square: - Used in guesses
+EMPTY_UNICODE = "\u200b"      # literally just an empty space
+
+ANTIDOTE_EMOJI = [
+    SYRINGE_EMOJI,
+    PILL_EMOJI,
+    HOURGLASS_EMOJI,
+    CROSSBONES_EMOJI,
+    ALEMBIC_EMOJI,
+]
```

Comment:
do we need to modify this? if not, what about using a tuple instead?

Reply:
changed all of these to tuples.